openpype/modules/deadline/plugins/publish/submit_publish_job.py

```diff
@@ -537,7 +547,16 @@ def _create_instances_for_aov(self, instance_data, exp_files):
                 # If expectedFile are absolute, we need only filenames
                 "stagingDir": staging,
                 "fps": new_instance.get("fps"),
-                "tags": ["review"] if preview else []
+                "tags": ["review"] if preview else [],
+                "colorspaceData": {
+                    "colorspace": colorspace,
+                    "config": {
+                        "path": additional_data["colorspaceConfig"],
```

Comment:
Isnt this absolute platform related path? I could track upstream coming form here https://github.com/ynput/OpenPype/pull/4340/files#diff-0f94b40dc0b348d7194c61cb01a57668eb3490d22cdbff89d37dd552287fccadR3422. Also presumably even platform absolute path could work with dirmaping, but just to make sure this works this way.

Reply:
Yeah, the path is platform specific. What are you suggesting to do about it?

Reply:
actually I am not sure about this. If the path is platform specific and the job is published from maya on windows, but the node will create review for it on linux, dirmapping won't work, am I right?

Reply:
Not sure how dirmapping works. Got any suggestions to make it work?

Reply:
on some places we are storing path "rootless", like "`{root[work]}/some/path/foo/bar`" and those paths are resolved on destination platform at the time of use.

Reply:
We can try an do a template, but the config can also be one of Maya's native configs in which case the template will just be the absolute path.